### ISSUE TITLE
Automate Kerchunk creation using sample dataset

### ISSUE THREAD

**Issue by jonmjoyce** [asascience-open/nextgen-dmac#26]
Create an Argo workflow to Kerchunk a dataset. Demonstrate that as new data becomes available (e.g. new forecast), the workflow will update the necessary kerchunk indexes for "best forecast" as well as the default kerchunk method.

Can use @mpiannucci Kerchunk example to start.

**Comment by jonmjoyce** [comment]
For now we can wait for David to check in his code. His code generates HRRR FMRC's when new forecasts are published.

**Comment by mpiannucci** [comment]
This has been prototyped in two different ways now. 

1. We implemented this on a kubernetes cluster, integrating argo-events with SQS messages to automate the creation of kerchunked DBOFS model data directly from NODD. (https://github.com/asascience-open/nextgen-dmac/tree/main/kerchunk)
2. Then we implemented the same logic in a separate stack using AWS SQS and Lambda functions, orchestrated with pulumi (https://github.com/asascience-open/nextgen-dmac/tree/main/cloud_aggregator)

This effort led to the the creation of the `ingest_tools` package which contains code for kerchunking NOS ROMS models and some tests to go along with them. This package is a starting point for creating a flexible python base for building cloud aggregation workflows in a common manner.
